src/rgw/rgw_admin.cc

```diff
@@ -9505,7 +9505,11 @@ int main(int argc, const char **argv)
     }
     const rgw::lua::context script_ctx = rgw::lua::to_context(*str_script_ctx);
     if (script_ctx == rgw::lua::context::none) {
-      cerr << "ERROR: invalid script context: " << *str_script_ctx << ". must be one of: preRequest, postRequest" <<  std::endl;
+      cerr << "ERROR: invalid script context: " << *str_script_ctx << ". must be one of: preRequest, postRequest, background" <<  std::endl;
```

Comment:
could you please take this opportunity to replace two consecutive spaces with a single one?